queasars/minimum_eigensolvers/evqe/evolutionary_algorithm/individual.py

```diff
@@ -298,7 +298,6 @@ def get_partially_parameterized_quantum_circuit(self, parameterized_layers: set[
         """
         n_qubits: int = self.layers[0].n_qubits
         circuit: QuantumCircuit = QuantumCircuit(n_qubits)
-        circuit.h(range(0, n_qubits))
```

Comment:
Since the default in all the new methods is `None` for the `initial_state_circuit` this effectively means that you switched EVQE from starting with a general super position to the zero state, was that intented?

Reply:
Yes this is intentional. As far as i can tell from the original paper, they do not prepend any state (therefore the default=None) and instead create a superposition by randomizing the parameters of the first layer of the individuals during the first population initialization. This of course necessitates me setting the layer randomization to True by default which i have done now.